esp/services/ws_workunits/ws_workunitsHelpers.cpp

```diff
@@ -3619,8 +3606,7 @@ void CWsWuFileHelper::writeToFileIOStream(const char* folder, const char* file,
     if (isEmptyString(file))
         throw MakeStringException(ECLWATCH_INVALID_INPUT, "Empty file name is not allowed to create FileIOStream.");
     VStringBuffer fileNameWithPath("%s%c%s", folder, PATHSEPCHAR, file);
-    CWsWuFileHelper helper(nullptr);
-    Owned<IFileIOStream> outIOS = helper.createIOStreamWithFileName(fileNameWithPath.str(), IFOcreate);
+    Owned<IFileIOStream> outIOS = createIOStreamWithFileName(fileNameWithPath.str(), IFOcreate);
```

Comment:
Should the createBufferedIOStreamWithFileName() be called here?

Reply:
My feeling is that since the write using the stream a few lines below is a single operation, buffering wouldn't improve performance.